src/components/node-list/node-list-row-list.js

```diff
@@ -4,6 +4,19 @@ import { connect } from 'react-redux';
 import NodeListRow, { nodeListRowHeight } from './node-list-row';
 import LazyList from '../lazy-list';
 
+// modify display of labels for modular pipelines to show nested relationship
+const getItemLabel = (item) => {
+  const layer = '・';
+  const whiteSpace = '&nbsp;&nbsp;&nbsp;&nbsp;';
+  if (item.type === 'modularPipeline') {
+    const levels = item.id.match(/\./g) ? item.id.match(/\./g).length : 0;
+    return `${whiteSpace
+      .repeat(levels)
+      .concat(layer.repeat(levels).concat(item.highlightedLabel))}`;
```

Comment:
Why are we wrapping this in a multiline string? It doesn't seem necessary to me.

```suggestion
    return whiteSpace
      .repeat(levels)
      .concat(layer.repeat(levels))
      .concat(item.highlightedLabel);
```